targettests/execution/state_init_err_runtime.cpp

```diff
@@ -8,11 +8,14 @@
 
 // clang-format off
 // Note: change |& to 2>&1| if running in bash
-// RUN: nvq++ %cpp_std %s -o %t --target quantinuum --emulate && %t |& FileCheck %s
+// RUN: nvq++ %cpp_std %s -o %t -target quantinuum -emulate && %t |& FileCheck %s
 // Note: change |& to 2>&1| if running in bash
-// RUN: nvq++ %cpp_std --enable-mlir --target remote-mqpu --remote-mqpu-url localhost:9999 %s -o %t && %t |& FileCheck %s
+// RUN: nvq++ %cpp_std --enable-mlir -target remote-mqpu --remote-mqpu-url localhost:9999 %s -o %t && %t |& FileCheck %s
 // clang-format on
 
+// FIXME! Requires init_state to be implemented via new argument synthesis
```

Comment:
This test checks for error message on quantum devices, ie verifies that it is indeed giving "unsupported " message. The message itself is probably changed between the old and the new synthesis, can the test to be updated with the new message instead of being disabled?

Reply:
Easy fix. Thanks.